### PR TITLE
Add Dynamed Plus

### PR DESCRIPTION
**Pull request by tf2manu994**
Note: This is the first non-news thing I've added. This one is a bit finicky, it only works once you close all its tabs, but thought it might be useful.

### CODE REVIEW

**Comment by tf2manu994** [comment]
The issue is it only works sometimes. If someone could do some more testing on when it works to delete cookies, that would be great. Just search anything on the site and click an article to enumerate your viewed article count.

**Comment by tf2manu994** [comment]
I need to add `ebsco.com` and `ebscohost.com` I think.

**Comment by tf2manu994** [comment]
Just tested, should be only `dynamed.com` and `www.dynamed.com`

File: manifest.json [hunk 1]
```diff
@@ -18,5 +18,5 @@
 		"page": "options.html"
 	},
     "permissions": [ "cookies", "<all_urls>", "storage", "webRequest", "webRequestBlocking"],
-	"version": "1.2.9"
+	"version": "1.2.12"
```
Comment (iamadamdev) [inline manifest.json]: change to 1.2.11

**Comment by tf2manu994** [comment]
@iamadamdev done

**Pull request merged**